core/domain/classifier_services.py

```diff
@@ -124,6 +124,57 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def train(exploration):
```

Comment:
@seanlip . So, I had a talk with Prasanna regarding this. He believes that trying to make this function asynchronous would be unnecessary since we will be shifting his train() to the VM in some time, where it does not need to be async. So, we could keep it like this here since it gets the pipeline flows and is in all aspects a better way than training in the learner view. Your suggestions?